### DISCUSSION TITLE
S3 Provisioner requires User credentials. It does not work with assumed Role credentials.

### DISCUSSION THREAD

**Discussion by FlorianSeidelAWS** [eclipse-edc/Connector#2457] [Ideas]
In AWS it is possible to assume IAM Roles to obtain credentials. AWS recommends to use IAM Roles for processes running in container, EC2 instances and so on. The current implementation of the S3 Provisioner requires that IAM user credentials are used. We should also allow IAM Roles. 

If the community agrees that this makes sense I volunteer to make the changes.

**Comment by jimmarino** [comment]
+1 you guys are definitely the subject matter experts!
